src/engines/aurora/kotorjadegui/listbox.cpp

```diff
@@ -248,6 +248,13 @@ void WidgetListBox::selectItemByWidgetTag(const Common::UString &tag) {
 	}
 }
 
+void WidgetListBox::selectItemByIndex(int index) {
+	if (index < 0 && index >= _items.size())
```

Comment:
```
src/engines/aurora/kotorjadegui/listbox.cpp: In member function ‘void Engines::WidgetListBox::selectItemByIndex(int)’:
src/engines/aurora/kotorjadegui/listbox.cpp:252:25: error: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<Common::UString>::size_type’ {aka ‘long unsigned int’} [-Werror=sign-compare]
  if (index < 0 && index >= _items.size())
                   ~~~~~~^~~~~~~~~~~~~~~~
```

I.e. add an `(uint)` to the second `index`.

Also, that should be a `||`, not an `&&`.